CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `setSinkId`, and the demo will not log an error in these cases.
 - [Demo] The meeting readiness checker no longer re-initializes the device output list
   after the user picked a device.
+- [Test] Fix voice focus integration/canary test.
```

Comment:
We dont need to do CHANGELOG for Test and Demo anymore.

Reply:
It's not required. But isn't it good to have?